### ISSUE TITLE
text changes on training basics screen

### ISSUE THREAD

**Issue by JTFouquier** [JTFouquier/mark2cure#3]
On the training basics page (http://mark2cure.org/training/basics/), enclose the indicated word/phrase in double quotes, and add a trailing period.  For example, we currently have:

```
Highlight words by clicking on them. Highlight the word Biomedical
```

Change to this:

```
Highlight words by clicking on them. Highlight the word "Biomedical".
```

----------------------------------------
- Bitbucket: https://bitbucket.org/sulab/mark2cure/issue/3
- Originally reported by: [Andrew Su](http://bitbucket.org/asu)
- Originally created at: 2014-12-02T19:59:35.450

**Comment by JTFouquier** [comment]
simple task so I'm going to make this a blocker...

----------------------------------------
Original comment by: [Andrew Su](http://bitbucket.org/asu)

**Comment by JTFouquier** [comment]
![](http://www.puff.me.uk/ss/KZWPQWwnr4Tqz.png)

----------------------------------------
Original comment by: [Max Nanis](http://bitbucket.org/x0xMaximus)

**Comment by JTFouquier** [comment]
closes #3  and slight css edits with blue bg class


→ <<cset 83f0bed400e0>>

----------------------------------------
Original comment by: [Max Nanis](http://bitbucket.org/x0xMaximus)
